src/javafundamentals/annotations/Ex01Annotation.java

```diff
@@ -4,6 +4,10 @@
  * Create your own annotation that must contain at least 3 types and 1 uses a default value.
  * Use the annotation to annotate this class (Ex01Annotation)
  */
-
+@MyAnnotation
 public class Ex01Annotation {
 }
+
+@interface MyAnnotation {
+    public String value() default "";
```

Comment:
This only contains 1 value! The exercise says 3! hehe

Reply:
Whoops, need to make sure i read the questions better.